cl/lib/model_helpers.py

```diff
@@ -90,10 +105,32 @@ def make_docket_number_core(docket_number: str | None) -> str:
     if district_m:
         return f"{district_m.group(1)}{int(district_m.group(2)):05d}"
 
-    bankr_m = re.search(appellate_bankr_d_num_regex, cleaned_docket_number)
-    if bankr_m:
-        # Pad to six characters because some courts have a LOT of bankruptcies
-        return f"{bankr_m.group(1)}{int(bankr_m.group(2)):06d}"
+    if bankr_n_core := make_appellate_bankr_number_core(cleaned_docket_number):
+        return bankr_n_core
+
+    return ""
+
+
+def make_scotus_docket_number_core(docket_number: str | None) -> str:
+    """Normalize SCOTUS docket numbers like 16A985.
+
+    :param docket_number: The docket number to condense.
+    :return: empty string if no change possible, or the condensed version if it
+    worked. Note that all values returned are strings. We cannot return an int
+    because that'd strip leading zeroes, which we need.
+    """
+    if not docket_number:
+        return ""
+
+    cleaned_docket_number = clean_docket_number(docket_number)
+
+    if bankr_n_core := make_appellate_bankr_number_core(cleaned_docket_number):
+        return bankr_n_core
+
+    scouts_a_m = re.search(scotus_d_a_num_regex, cleaned_docket_number)
+    if scouts_a_m:
+        year, serial = scouts_a_m.groups()
+        return f"{year}A{int(serial):05d}"
 
     return ""
```

Comment:
We should add tests for this helper, similar to the ones in `test_making_docket_number_core`.